.travis/after_success.sh

```diff
@@ -24,6 +24,11 @@ if [[ "$TRAVIS_BRANCH" == "master" ]]; then
   echo "Running npm prune"
   npm prune
 
+  # Without this releasing is broken as the dependency is not installed
+  # with yarn. Switching to npm does not seem to be a sane alternative.
+  echo "Installing buffer-shims"
+  npm install buffer-shims
```

Comment:
Is adding `buffer-shims` as an explicit dependency with a pinned version not an option?

Reply:
Yeah maybe. We didn't want to have to manage it as another dependency. Maybe installing a pinned version here, the one the package underneath wants is cool